app/controllers/concerns/authenticate/by_bearer_token.rb

```diff
@@ -0,0 +1,28 @@
+module Authenticate::ByBearerToken
+
+  private
+
+  def find_client_by_bearer_token
+    authenticate_with_http_token do |double_token, options|
+      client, client_token = parse_double_token(double_token)
```

Comment:
If the `Authentication` isn't present for any reason, then this returns `nil` for `client` and an error is raised when trying to compare `client.token` (client being `nil`) with `client_token`. We could change the condition to be `if client && client_token && ...` perhaps.

Reply:
Good catch. I fixed it and added a test to catch it.